packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -22,15 +22,30 @@ import { getTsConfigs } from '../project'
 
 export const generateTypeDefGraphQLApi = async () => {
   const filename = path.join(getPaths().api.types, 'graphql.d.ts')
+  const prismaModels = getPrismaModels()
+  const prismaImports = Object.keys(prismaModels).map((key) => {
```

Comment:
Moved importing Prisma types here, because we now use a "custom mapper" generic `MergePrismaWithSdlTypes<>` in the mapper object.